data/creatures.js

```diff
@@ -0,0 +1,164 @@
+const creatures = [
```

Comment:
because you created this data file you could just import in the test but all the app.js if you wanted to.